packages/suite/src/actions/wallet/__tests__/coinjoinClientActions.test.ts

```diff
@@ -0,0 +1,96 @@
+import { combineReducers, createReducer } from '@reduxjs/toolkit';
+import { configureMockStore } from '@suite-common/test-utils';
+
+import { accountsReducer } from '@wallet-reducers';
+import { coinjoinReducer } from '@wallet-reducers/coinjoinReducer';
+import { onCoinjoinRoundChanged } from '../coinjoinClientActions';
+import * as fixtures from '../__fixtures__/coinjoinClientActions';
+
+jest.mock('@trezor/connect', () => global.JestMocks.getTrezorConnect({}));
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const TrezorConnect = require('@trezor/connect').default;
+
+const DEVICE = global.JestMocks.getSuiteDevice({ state: 'device-state', connected: true });
+
+const rootReducer = combineReducers({
+    suite: createReducer(
+        {
+            locks: [],
+            device: DEVICE,
+            settings: {
+                debug: {},
+            },
+        },
+        {},
+    ),
+    devices: createReducer([DEVICE], {}),
+    modal: () => ({}),
+    wallet: combineReducers({
+        coinjoin: coinjoinReducer,
+        accounts: accountsReducer,
+    }),
+});
+
+type State = ReturnType<typeof rootReducer>;
+type Wallet = Partial<State['wallet']> & { devices?: State['devices'] };
+
+const initStore = ({ accounts, coinjoin, devices }: Wallet = {}) => {
+    // TODO: didn't found better way how to generate initial state or pass it dynamically to rootReducer creator
+    const preloadedState: State = JSON.parse(
```

Comment:
looks OK for usage in tests

Maybe if this partial pre-filling of redux store will be more common in tests, we can add some support of this directly into `configureMockStore`, like first `deepCopy` of returned state from `rootReducer` and then `deepMerge`

Reply:
ok, so im leaving it as is, dont have time for detours ;)